.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] As suggested by the previous reviewer. please list `node_modules/` in your .gitignore file also as this is a project reequirement.

Reply:
- [ ] You are almost there! However, as the previous reviewers said, it would be great to add `node_modules/` to your `.gitignore` file, as this is a requirement for the activity.